tools/astropytools/fitsinfo.xml

```diff
@@ -1,15 +1,15 @@
-<tool id="fitsinfo" name="astropy fitsinfo" version="0.1.0+galaxy0" profile="21.05">
+<tool id="astropy_fitsinfo" name="astropy fitsinfo" version="0.1.0+galaxy0" profile="21.05">
     <requirements>
         <requirement type="package" version="5.1">astropy</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
-        fitsinfo '$input1' | grep -vi filename > '$output1' 
+        fitsinfo '$input' | grep -vi filename > '$output' 
     ]]></command>
     <inputs>
-        <param type="data" name="input1" format="fits" />
+        <param type="data" name="input" format="fits" />
```

Comment:
label and help could be added and the overall tool help could be improved.

Its not very useful to explain the parameters, for that you have label and help on the params. Instead write what the tool is doing, explain the output etc ...